syntax = "proto3";
import "config.proto";

message AllLists {
  repeated DeviceConfig product_list = 1;
  int32 checksum = 2;
}